tests/test_find_noisy_channels.py

```diff
@@ -1,155 +1,212 @@
+    nd = NoisyChannels(raw_tmp, do_detrend=False)
     nd.find_bad_by_deviation()
-    assert rand_chn_lab in nd.bad_by_deviation
-
-    # Test for correlation between EEG channels
-    raw_tmp = raw.copy()
-    m, n = raw_tmp._data.shape
-    rand_chn_idx = int(rng.randint(0, m, 1))
-    rand_chn_lab = raw_tmp.ch_names[rand_chn_idx]
-    # Use cosine instead of sine to create a signal
-    low = 10
-    high = 30
-    n_freq = 5
-    signal = np.zeros((1, n))
-    for freq_i in range(n_freq):
-        freq = rng.randint(low, high, n)
-        signal[0, :] += np.cos(2 * np.pi * raw.times * freq)
-    raw_tmp._data[rand_chn_idx, :] = signal * 1e-6
-    nd = NoisyChannels(raw_tmp, random_state=rng)
-    nd.find_bad_by_correlation()
-    assert rand_chn_lab in nd.bad_by_correlation
-    bad_by_correlation_orig = nd.bad_by_correlation  # save for dropout tests
-
-    # Test for channels with signal dropouts (reuse data from correlation tests)
-    dropout_idx = rand_chn_idx - 1 if rand_chn_idx > 0 else 1
-    # Make 2nd and 4th quarters of the dropout channel completely flat
-    raw_tmp._data[dropout_idx, :int(n/4)] = 0
-    raw_tmp._data[dropout_idx, int(3*n/4):] = 0
-    # Run correlation and dropout detection on data
-    nd = NoisyChannels(raw_tmp, random_state=rng)
-    nd.find_bad_by_correlation()  # also does dropout detection
-    # Test if dropout channel detected correctly
-    assert raw_tmp.ch_names[dropout_idx] in nd.bad_by_dropout
-    # Test if correlations still detected correctly
-    bad_orig_plus_dropout = bad_by_correlation_orig + nd.bad_by_dropout
-    same_bads = set(nd.bad_by_correlation) == set(bad_by_correlation_orig)
-    same_plus_dropout = set(nd.bad_by_correlation) == set(bad_orig_plus_dropout)
-    assert same_bads or same_plus_dropout
-
-    # Test for high freq noise detection
-    raw_tmp = raw.copy()
-    m, n = raw_tmp._data.shape
-    rand_chn_idx = int(rng.randint(0, m, 1))
-    rand_chn_lab = raw_tmp.ch_names[rand_chn_idx]
-    # Use freqs between 90 and 100 Hz to insert hf noise
-    signal = np.zeros((1, n))
-    for freq_i in range(n_freq):
-        freq = rng.randint(90, 100, n)
-        signal[0, :] += np.sin(2 * np.pi * raw.times * freq)
-    raw_tmp._data[rand_chn_idx, :] = signal * 1e-6
-    nd = NoisyChannels(raw_tmp, random_state=rng)
+    assert nd.bad_by_deviation == [raw_tmp.ch_names[high_dev_idx]]
+
+    # Make the signal for a different channel have a very low amplitude
+    low_dev_idx = (high_dev_idx - 1) if high_dev_idx > 0 else 1
+    raw_tmp._data[low_dev_idx, :] *= low_dev_factor
+
+    # Test detection of abnormally low-amplitude channels
+    # NOTE: The default z-score threshold (5.0) is too strict to allow detection
+    # of abnormally low-amplitude channels in some datasets. Using a relaxed Z
+    # threshold of 3.29 (p < 0.001, two-tailed) until a better solution is found.
+    nd = NoisyChannels(raw_tmp, do_detrend=False)
+    nd.find_bad_by_deviation(deviation_threshold=3.29)
```

Comment:
Could we document this in the MatPREP differences document? And should we wrap the original threshold of 5 in a `matlab_strict`  conditional? What do you think? I am all for improving Matprep where it makes sense (see also your issue https://github.com/VisLab/EEG-Clean-Tools/issues/26, and the median problem for which we should open an issue at some point, etc.) ... BUT for a 0.4 release I think it'd be great to get MatPrep equivalence, and not go beyond already (at least not without a matlab_strict option).

Reply:
I'm just modifying the test behaviour here rather than changing anything about NoisyChannels itself, I figured that a proper `matlab_strict` wrapper and explanation would be part of whatever PR ends up properly addressing #83 (which I'm fine leaving for after 0.4, I just wanted to make note of the issue as soon as I noticed it).

Reply:
okay, makes sense :+1: